psi-ms.obo

```diff
@@ -1,10 +1,10 @@
 format-version: 1.2
-data-version: 4.1.65
+data-version: 4.1.66
 date: 04:12:2021 00:00
 saved-by: Eric Deutsch
 auto-generated-by: OBO-Edit 2.3.1
-import: http://ontologies.berkeleybop.org/pato.obo
-import: http://ontologies.berkeleybop.org/uo.obo
+import: http://purl.obolibrary.org/obo/pato.obo
```

Comment:
Much better!

Even better would be to use the OWL PURL. As you are hand editing the obo file I don't think this will negatively impact you.

Reply:
Would that mess up people using OBO-Edit?